osrf/vrx
# Update the allowed region for the thrusters

See the discussion in issue #398.

I updated the technical guide documentation including:

* Updated image showing the allowed regions for placing thrusters.
* Updated front thruster bounding boxes positions. The reason for this change is to make sure that the default `X` configuration is within the bounds.

How to test it?

* Follow the example 1 section of [this tutorial](https://github.com/osrf/vrx/wiki/tutorials-Creating-a-custom-WAM-V-Thruster-and-Component-Configuration-For-Competition).

* Replace `thruster_config.yaml` with the default `X` configuration:

```
engine:
  - prefix: "left_rear"
    position: "-2.373776 1.027135 0.318237"
    orientation: "0.0 0.0 0.785"
  - prefix: "right_rear"
    position: "-2.373776 -1.027135 0.318237"
    orientation: "0.0 0.0 -0.785"
  - prefix: "left_front"
    position: "1.6 0.7  0.25"
    orientation: "0.0 0.0 -0.785"
  - prefix: "right_front"
    position: "1.6 -0.7 0.25"
    orientation: "0.0 0.0 0.785"
```

* Rerun `generate_wamv.launch` and verify that there are no errors and the WAM-V loads properly.

Thanks to @Tinker-Twins for the great discussion.
